ci_framework/roles/ci_setup/tasks/packages.yml

```diff
@@ -24,8 +24,9 @@
       register: oc_version
       environment:
         PATH: "{{ cifmw_path }}"
-      ansible.builtin.command:
-        cmd: "oc version -o yaml"
+      ci_script:
```

Comment:
not sure `ci_script` adds any value here. Getting the whole reproducer + log file just for this `oc version` isn't worth it imho

Reply:
Got it. I won't change for simple commands.